src/contract-verification/sourcify/integration-modes.md

```diff
@@ -32,8 +32,13 @@ The Sourcify snapshot is provided as a nginx image at: <https://hub.docker.com/r
 
 You can run it with:
 
-```shell
-docker run --rm -d -p 3006:80 --name sourcify-snapshot otterscan/sourcify-snapshot:2021-09
+```sh
+docker run \
+  --rm \
+  -d \
+  -p 3006:80 \
+  --name sourcify-snapshot \
+  otterscan/sourcify-snapshot:2021-09
```

Comment:
I think some of the Docker arguments should appear on a single line, like `--rm -d -p 3006:80`.

Reply:
I agree that it is a little ugly (and I thought about it), but consistent with having each flag (+its args) per line.

so I think we should keep it this way (at least for now)